### DISCUSSION TITLE
Table select all includes disabled checkboxes

### DISCUSSION THREAD

**Discussion by cdecinkoKnight** [quasarframework/quasar#16250] [General - Components / Directives / etc] [answered]
I have a q-table set with selection="multiple". This allows selecting all visible rows. I have a condition that disables some rows. I set the :disable property on those checkboxes to true. 

When clicking the top checkbox to select all I expect only the rows with their checkbox not disabled to be included in the selected array.  i.e. I have five rows showing, two of those rows have their checkbox disabled. But when clicking select all, the array contains all 5 rows.

**Comment by metalsadman** [comment, marked as answer]
@cdecinkoKnight use `selected` event 
<img width="467" alt="image" src="https://github.com/quasarframework/quasar/assets/31038838/683e144c-2147-4f4d-a995-71326f3dbaf0">

using this example https://quasar.dev/vue-components/table#example--selection-cell-slots
https://codepen.io/metalsadman/pen/rNoOjPN?editors=1010

**Reply by gjimenez-gc** [reply]
Hi, I have been playing around with the q-table, is there a way to disable specific rows in selection mode "single"?

**Reply by metalsadman** [reply]
https://codepen.io/metalsadman/pen/ZEgQwdm?editors=101

**Reply by gjimenez-gc** [reply]
Thank you very much, been banging my head for days on this!

**Reply by gjimenez-gc** [reply]
@metalsadman sorry to bother you again, is there a way to change the style, I tried like that but is not working :/ 

```
 <template v-slot:body-cell-selection="props" class="sticky--col">
                    <q-td :props="props" class="sticky--col">
                        <q-checkbox class="sticky--col" v-model="props.row.selected"
                            @update:model-value="toggleSelection(props)" />
                    </q-td>
        </template>
```

**Reply by metalsadman** [reply]
try the css implemented here https://quasar.dev/vue-components/table#example--sticky-first-column
